toolkit/locales/en-US/chrome/global/aboutSupport.ftl

```diff
@@ -45,6 +65,13 @@ aboutSupport-appBasicsMultiProcessSupport = Multiprocess Windows
 aboutSupport-appBasicsSafeMode = Safe Mode
 aboutSupport-appBasicsProfiles = Profiles
 
+# This string displays the number of open multiprocess (e10s) windows compared
+# to the overall number of open windows, and the current status of e10s in the
+# browser.
+# $remote (Number) - Number of multiprocess windows
+# $total (Number)  - Total number of open windows
+# $status (String) - Current status of e10s
```

Comment:
Ok, so you'd end up with sth like this:

```
# $status (Number) - Numerical index used to identify the current status of e10s.
#  The status message corresponding to the index will be displayed in the UI, e.g. “Enabled by user” for 0.
aboutSupport-multiProcessWindows = { $remote }/{ $total } ({ $status ->
  [0] Enabled by user
  [1] Enabled by default
}
```

it feels like the doc string repeats what's in the entity itself? Is that what you want?

Alternative in my head would be sth like this:

```
# $statusCode (Number) - code of the status of e10s in the browser. Used to select description part of the string.
aboutSupport-multiProcessWindows = { $remote }/{ $total } ({ $status ->
  [0] Enabled by user
  [1] Enabled by default
}
```

What do you think?